app/helpers/custom_components_helper.rb

```diff
@@ -19,4 +23,10 @@ def render_embedded_app(fragment)
       return render template: 'apps/child_birth_app/picking_up_protocol/start', layout: 'layouts/embedded_app'
     end
   end
+
+  def render_notification_subscription(fragment)
+    @types = fragment[:types].split(',')
+    @extra_attributes = fragment.attributes.except('types').map { |k,v| [k.to_sym, v.value] }.to_h
```

Comment:
Toto teda mozeme dat von.

Reply:
Ak si sa `@extra_attributes` rozhodol ponechat, sprav z toho a z `@types` prosim local variables, t.j. `extra_attributes` a `types`

Reply:
Ako ich potom pouzijem vo view mimo helpera?

Reply:
Vies pouzit toto? https://guides.rubyonrails.org/layouts_and_rendering.html#passing-local-variables

Reply:
To je super feature, o tom som nevedel